src/runtime/providers/cloudimage.ts

```diff
@@ -55,6 +55,12 @@ export const getImage: ProviderGetImage = (src, {
     cdnURL = `https://${token}.cloudimg.io/${apiVersion}`
   }
 
+  if (src.startsWith('http')) {
```

Comment:
Can we perhaps use a more precise test? for example, this would match `http-some-file.png`.